pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3141,15 +3141,6 @@
         "xamarinmac20": "Any",
         "xamarintvos10": "Any",
         "xamarinwatchos10": "Any"
-      },
```

Comment:
Do not remove this.  Only remove the mapping for 4.6.0 since that won't be shipping.